kbase-extension/scss/partials/_bulkImportCell.scss

```diff
@@ -221,6 +221,15 @@ $button-outdated: rgb(247, 155, 34);
   &__param_container {
     margin: 3px;
   }
+
+  &__input_control select {
+    margin-left: 0;
+  }
+
+  &__cell_label {
+    margin-bottom: 0;
+    padding-top: 8px;
```

Comment:
i'm not sold on this approach, but it was the fastest way to achieve more reasonable spacing of the fields and labels ahead of the demo